docs/jwst/outlier_detection/outlier_detection_coron.rst

```diff
@@ -1,22 +1,21 @@
 .. _outlier-detection-coron:
 
-Outlier Detection for Coronagraphic Data
-========================================
+Coronagraphic Data
+==================
 
 This module serves as the interface for applying ``outlier_detection`` to coronagraphic
-image observations.
-
-Specifically, this routine performs the following operations:
+image observations. A :py:class:`~jwst.datamodels.CubeModel` serves as the basic format
+for all processing performed by this step. This routine performs the following operations:
 
 #. Extract parameter settings from input model and merge them with any user-provided values.
    See :ref:`outlier detection arguments <outlier_detection_step_args>` for the full list
    of parameters.
 
-#. Convert input data, as needed, to make sure it is in a format that can be processed.
-   A :py:class:`~jwst.datamodels.CubeModel` serves as the basic format for all processing
-   performed by this step.
+#. Do not attempt resampling; data are assumed to be aligned and have an identical WCS.
+   This is true automatically for a CubeModel.
 
-#. Create a median image preserving the spatial dimensions of the cube.
+#. Create a median image over the `groups` (exposures, planes of cube) axis,
+   preserving the spatial (x,y) dimensions of the cube.
```

Comment:
I can't mark this in the right place since you didn't edit it, but in the sentence below, resampling is referenced and shouldn't be.